### ISSUE TITLE
undefined method 'starts_with?' for String

### ISSUE THREAD

**Issue by darena-mdsol** [mdsol/dice_bag#81]
https://github.com/mdsol/dice_bag/blob/817ec00de5e0d9cd4657598e5b6fafb85e9f9907/lib/dice_bag/available_templates.rb#L30

while trying to generate templates for mauth-client gem using `bundle exec rake config:generate_for_gems mauth-client, I get this error:

>rake aborted!
NoMethodError: undefined method `starts_with?' for #<String:0x007fb996262300>
Did you mean?  start_with?

Rails 5 api only mode is installed.

**Comment by clabrunda** [comment]
Were you able to get this working by changing it to start_with?  I would be happy to review and merge such a change, as long as it also includes:
```
# Dice bag requires ruby 1.8.7 or greater.
s.required_ruby_version = '>= 1.8.7'
```
added to dice_bag.gemspec.

**Comment by darena-mdsol** [comment]
I did exactly that and the task ran successfully. I make the change and submit a PR.

**Comment by jcarres-mdsol** [comment]
Solved by #82
